platform.md

```diff
@@ -182,13 +182,13 @@ The platform SHOULD ensure that:
 
 The platform MUST ensure that:
 
-- The image config's `User` field is set to a user with the same user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID and primary group [†](README.md#operating-system-conventions)GID/[‡](README.md#operating-system-conventions)SID as in the build image.
 - The image config's `Label` field has the label `io.buildpacks.stack.id` set to the stack ID.
 - The image config's `Label` field has the label `io.buildpacks.stack.mixins` set to a JSON array containing mixin names for each mixin applied to the image.
 - The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`).
 
 The platform SHOULD ensure that:
 
+- The image config's `User` field is set to a user with a **DIFFERENT** user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID as the build image.
```

Comment:
Do we want to say anything about the group ID?

Reply:
I purposely left it out since the RFC recommended they could either be the same as the build stack or different depending on operator use cases, which seemed like too loose to go into the platform spec and instead maybe should go in the docs for operators?

Reply:
In the RFC it's a "recommendation" which to me means we should be using "SHOULD" vs. "MUST".

```suggestion
- The image config's `User` field is SHOULD be set to a user with a **DIFFERENT** user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID as the build image.
```

Reply:
The whole set of clauses here are under a `SHOULD` (Line 189/190) which is why I left it out. :)

Reply:
@samj1912 oh right, good call. :+1: